tapback/frontend.c

```diff
@@ -509,30 +504,7 @@ tapback_backend_handle_otherend_watch(backend_t *backend,
      */
 	s = tapback_xs_read(device->backend->xs, XBT_NULL, "%s",
 			device->frontend_state_path);
-    if (!s) {
-        err = errno;
-		/*
-         * If the front-end XenBus node is missing, the XenBus device has been
-         * removed: remove the XenBus back-end node.
-		 */
-		if (err == ENOENT) {
-            err = asprintf(&_path, "%s/%s/%d/%d", XENSTORE_BACKEND,
-                    device->backend->name, device->domid, device->devid);
-            if (err == -1) {
-                err = errno;
-                WARN(device, "failed to asprintf: %s\n", strerror(err));
-                goto out;
-            }
-            err = 0;
-            if (!xs_rm(device->backend->xs, XBT_NULL, _path)) {
-                if (errno != ENOENT) {
-                    err = errno;
-                    WARN(device, "failed to remove %s: %s\n", path,
-                            strerror(err));
-                }
-            }
-		}
-    } else {
```

Comment:
Indentation (from original code it seems, mixing tabs and spaces).

That block looks ok though: If the backend no longer finds the frontend, should it not remove its Xenstore entries?

Reply:
Ack, will fix the indentation.
That block of code removing the Xenstore entries is never triggered and hence redundant.